src/ansible_navigator/_version.py

```diff
@@ -1,7 +1,6 @@
-""" version
+"""Version information for ansible-navigator.
 
 Note:
```

Comment:
Looks like this could be an RST `.. note::` admonition.

Reply:
done